pkgs/desktops/gnome-3/extensions/paperwm/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-paperwm";
+  version = "34.3";
+
+  src = fetchFromGitHub {
+    owner = "paperwm";
+    repo = "PaperWM";
+    rev = version;
+    sha256 = "1qry75f696pgmd9yzqvwhq5h6cipin2fvk7h881g29cjcpxim37a";
+  };
+
+  uuid = "paperwm@hedning:matrix.org";
+
```

The build phase doesn't do anything as upstream ships compiled schemas.
```suggestion

   dontBuild = true;
```

---

Updated.